recipe/meta.yaml

```diff
@@ -71,7 +72,8 @@ test:
   commands:
     - ipython -h
     - ipython3 -h
-    - NOSE_EXCLUDE="test_history|recursion" iptest  # [unix and not (aarch64 or ppc64le)]
+    # [unix and not (aarch64 or ppc64le)]
```

Comment:
what does moving this here do? should we just make a `run_test.py` to encapsulate all of this?